dev/update-examples.sh

```diff
@@ -22,9 +22,16 @@ cd examples/
 for d in $(printf '%s\n' */ | sort -V); do
   example=${d%/}
   # For each example, copy the README into the source of the Example docs
-  [[ $example = doc ]] || cp $example/README.md $ROOT/examples/doc/source/$example.md 2>&1 >/dev/null
+  [[ $example != doc ]] && cp $example/README.md $ROOT/examples/doc/source/$example.md 2>&1 >/dev/null
+  # For each example, copy all images of the _static folder into the examples
+  # docs static folder
+  [[ $example != doc ]] && [ -d "$example/_static" ] && {
+    cp $example/_static/**.jpg $ROOT/examples/doc/source/_static/ 2>/dev/null || true
+    cp $example/_static/**.png $ROOT/examples/doc/source/_static/ 2>/dev/null || true
+    cp $example/_static/**.jpeg $ROOT/examples/doc/source/_static/ 2>/dev/null || true
```

Comment:
```suggestion
    cp $example/_static/**.{jpg,png,jpeg} $ROOT/examples/doc/source/_static/ 2>/dev/null || true
```
@charlesbvll I think this should also work and would be slightly less verbose.